dbt_platform_helper/utils/validation.py

```diff
@@ -461,11 +484,21 @@ def validate(self, data, _is_conditional_schema=True):
                 Optional("deletion_policy"): DELETION_POLICY,
                 Optional("plan"): OPENSEARCH_PLANS,
                 Optional("volume_size"): int,
+                Optional("ebs_throughput"): int,
+                Optional("ebs_volume_type"): str,
+                Optional("instance"): str,
+                Optional("instances"): int,
+                Optional("master"): bool,
+                Optional("es_app_log_retention_in_days"): int,
+                Optional("index_slow_log_retention_in_days"): int,
+                Optional("audit_log_retention_in_days"): int,
+                Optional("search_slow_log_retention_in_days"): int,
```

Comment:
Are these loads of additional configuration options we have added that we didn't have before?

Reply:
Yes.